src/Middleware/CORS/src/Infrastructure/CorsPolicy.cs

```diff
@@ -30,7 +30,7 @@ public bool AllowAnyHeader
         {
             get
             {
-                if (Headers == null || Headers.Count != 1 || Headers.Count == 1 && Headers[0] != "*")
+                if (Headers == null || Headers.Count != 1 || Headers[0] != "*")
```

Comment:
Couldn't the Count be 0?

Reply:
It could, but 0 is not 1, so would fall out of the `if` and not call the indexer?